lab_1/start.py

```diff
@@ -19,6 +20,9 @@
             file_to_read:
         unknown_text = file_to_read.read()
 
+    with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'la.txt'), 'r', encoding='utf-8') as file_to_read:
+        la_text = file_to_read.read()
```

Comment:
why there is no logic here?
you need to classify a text

Reply:
Start.py is done but I'm not sure whether I did everything correct. Could you please check it for me one more time? Thank you in advance!